### PR TITLE
rossanodan-issue-32: work on CONTRIBUTING guidelines

### PR DESCRIPTION
**Pull request by rossanodan**
Issue https://github.com/EddieJaoudeCommunity/EddieBotUI/issues/32

### This pull request contains
- Changes to `CONTRIBUTING.md` file. I added some details about how to contribute and a draft coding style guide

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+- **Custom issue**
+  Custom template, use this wisely.
+
+## Coding Style  
+* 2 spaces for indentation rather than tabs
+* 80 character line length
+* Run `npm run lint` to conform to our lint rules
+
+## License
+By contributing to Draft.js, you agree that your contributions will be licensed
```
Comment (BOLT04) [inline CONTRIBUTING.md]: I think it should be our repo's name here right 😅 ?
```suggestion
By contributing to EddieBotUI, you agree that your contributions will be licensed
```

**Review by BOLT04 — COMMENTED** [review]
Looks good, just one comment 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (rossanodan) [reply CONTRIBUTING.md]: @BOLT04 yes, I also forgot to mention Draft.js for the template!! I will update the PR tomorrow 💪🏻

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -34,9 +32,12 @@ There are three types of issue templates:
...
+By contributing to EddieBotUI, you agree that your contributions will be licensed
+under [MIT license](./LICENSE).
+
+## Need help?
+For **help**, **support** or **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) on Discord.
 
 ---
 
-For **help**, **support** or **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) 🚑.
+Special thanks go to the [Draft.js team](https://github.com/facebook/draft-js) which inspired us with this template.
```
Comment (eddiejaoude) [inline CONTRIBUTING.md:43]: Is this meant to be `Draft.js`?

Comment (rossanodan) [reply CONTRIBUTING.md:43]: Yes @eddiejaoude, I used their template. Clean and with essential things. We can simply add things to it :)

Comment (eddiejaoude) [reply CONTRIBUTING.md:43]: Ok 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**